proposals/052-cryptography-leg-1.md

```diff
@@ -0,0 +1,439 @@
+
+- Frequent (2-3x weekly) activity update on devlog
+- Weekly major update that includes github push
+- Monthly report that summarizes what specific items have been worked on, what items will be worked on, and what challenges have arisen, if any.
+
+### Deliverables
+
+The following are the intended deliverables of this proposal:
+
+- `botan-bindings`
+- `botan-low`
+- `botan`
+
+Each library should be beta- or / production-ready, and submitted to hackage as a candidate or accepted package. 
+
+Each library should have:
+
+- Installation instructions for the Botan library
+- Documentation
+- Tutorials and examples
+- Tracked issues
+- Unit tests
+- CI tests for Linux, MacOS, and Windows
+
+The official repo will be owned by the [Haskell Foundation](https://haskell.foundation/), and maintained by the [Haskell Cryptography Group](https://github.com/haskell-cryptography), with myself as the current maintainer. Another member of the Haskell Cryptography Group, or a member of the Haskell Foundation, will be selected to hold backup keys and permissions to any website, server, and official repositories as needed, in order to give project members access and to reduce any conflicts about ownership or maintenance.
+
+## Budget
+
+This proposal presents a minimum budget of $7000 USD per month, for one full-time engineer, for a minimum duration of 3 months, for an estimated total cost of $21,000 USD. This budget is based on cost-of-living and industry experience, and will cover housing, food, bills, taxes, and other life necessities for one engineer, as well as any project necessities such website and server hosting. This budget is roughly equivalent to $40 / hr or $84k / yr at full-time of 40 hrs / wk. Industry rates for an engineer of the necessary skill are on average considerably higher, and so we consider this budget to be reasonable. The exact legal contract / arrangement is left to the Haskell Foundation, and this timeline and budget may be extended at their discretion under the same terms.
+
+## Stakeholders
+
+### Positive stakeholders
+
+Several parties stand to gain from the implementation of this proposal.
+
+- Leon Dillinger (Myself)
+
+As the intended engineer and member of the Haskell Cryptography Group carrying out the work, I have already invested significant effort in this project in order to establish its viability, and will benefit from this proposal in the form of being the recipient of the funding.
+
+- [Haskell Foundation](https://haskell.foundation/)
+
+As the intended source of funding, the Haskell Foundation would be invested in seeing that its funding is spent effectively.
+
+- [Haskell Cryptography Group ](https://github.com/haskell-cryptography)
+
+As the intended owner of the project, this project will be new maintenance burden on the Haskell Cryptography Group.
```

Comment:
This probably needs rephrasing, given that HF is now supposed to become an owner.

Reply:
"This project will be new maintenance burden on the Haskell Cryptography Group, which is projected to be the maintainer designated by the Haskell Foundation"